schema.graphql

```diff
@@ -26,7 +30,10 @@ type Extrinsic @jsonField {
 
 type HistoryElement @entity {
   id: ID!
-  timestamp: String!
+  blockNumber: Int!
+  extrinsicIdx: Int,
+  extrinsicHash: String
+  timestamp: BigInt!
   address: String!
```

Comment:
Let's add index to this field

Reply:
Maybe it does make sense to create a multi-column index, if possible? It should be more efficient than two single-column indices (given that we'll use both in the same query)

Reply:
@valentunn  What do you mean? I can't find about multi-column index in docs https://doc.subquery.network/create/graphql.html#indexing-by-non-primary-key-field